prod-refractr.yml

```diff
@@ -768,3 +768,9 @@ refracts:
 
 - mozilla-hub.atlassian.net/: jira.mozilla.com
   status: 302
+
+  # bug 1660250
+- dsts:
```

Comment:
I don't have a huge amount of confidence, but have you tried just a simple redirect without srcs/dsts as seperate items?  I believe this example shows that it'll work as we expect - https://github.com/mozilla-it/refractr/blob/c4f80093a359f801f5564054372ee32f69080600/prod-refractr.yml#L287 and feels a bit cleaner.

Reply:
Yah, I think that does the right thing and is easier to understand.
```bash
170 server {
171     server_name techspeakers.mozilla.org;
172     add_header Strict-Transport-Security "max-age=60; includeSubDomains" always;
173     return 301 https://community.mozilla.org/en/groups/tech-speakers/;
174 }   
```